src/views/Player.vue

```diff
@@ -174,6 +174,9 @@ export default defineComponent({
       (newValue) => {
         if (newValue == numQuestions.value) {
           state.isScorecardShown = true;
+          if (!state.hasQuizEnded && !isQuizAssessment.value) {
```

Comment:
What is the second check for in the `if` statement?

Reply:
`isQuizAssessment` is set to `True` if quiz is assessment format. We are checking if its not an assessment (that is, is it a homework). I think we should probably make a `isQuizHomework` computed element too later on.